website/docs/integrations/atlantis.md

```diff
@@ -149,23 +161,498 @@ workflows:
 
-1. `atmos terraform generate backends --format=backend-config|hcl`
-2. `atmos terraform generate varfiles`
+Configuring the Atlantis Integration in the `settings.atlantis` sections in the stack configs has the following advantages:
 
-Make sure that the resulting files are committed back to VCS (e.g. `git add -A`) and push'd upstream. That way Atlantis will trigger on the "affected
-files" and propose a plan.
+- The `settings` section is a first class section in Atmos (similar to `vars`). It participates in deep-merging and in the inheritance chain. It can
+  be defined and overridden at any level (organization/namespace, OU/tenant, region/environment, account/stage, base component, component). You can
+  define the base settings at the org, tenant or account level, and then override some settings at the component level, making the whole configuration
+  DRY
+
+- When executing the `atmos atlantis generate repo-config` command, you don't need to pass the `--config-template` and `--project-template` flags to
+  specify which config and project templates to use. Instead, Atmos will get this information from the `settings.atlantis` section
 
-### Example GitHub Action
+- When executing the `atmos describe component <component> -s <stack>` command, you will see the configured Atlantis Integration in the outputs. For
+  example:
+
+  ```yaml title="atmos describe component test/test-component-override -s tenant1-ue2-dev"
+    atmos_component: test/test-component-override
+    atmos_stack: tenant1-ue2-dev
+    component: test/test-component
+    settings:
+      atlantis:
+        config_template:
+          allowed_regexp_prefixes:
+          - dev/
+          automerge: false
+          delete_source_branch_on_merge: false
+          parallel_apply: false
+          parallel_plan: true
+          version: 3
+        config_template_name: config-1
+        project_template:
+          apply_requirements:
+          - approved
+          autoplan:
+            enabled: true
+            when_modified:
+            - '**/*.tf'
+            - varfiles/$PROJECT_NAME.tfvars.json
+          delete_source_branch_on_merge: false
+          dir: '{component-path}'
+          name: '{tenant}-{environment}-{stage}-{component}'
+          terraform_version: v1.3
+          workflow: workflow-1
+          workspace: '{workspace}'
+        project_template_name: project-1
+        workflow_templates:
+          workflow-1:
+            apply:
+              steps:
+              - run: terraform apply $PLANFILE
+            plan:
+              steps:
+              - run: terraform init
+              - run: terraform workspace select $WORKSPACE || terraform workspace new $WORKSPACE
+              - run: terraform plan -out $PLANFILE -var-file varfiles/$PROJECT_NAME.tfvars.json
+      spacelift:
+        protect_from_deletion: true
+        stack_destructor_enabled: false
+        workspace_enabled: true
```

Comment:
If it's not relevant to the Atlantis configuration/integration, we should not include it here. 

```suggestion
```